Automattic/wp-calypso
# Stepper: Fix navigation UI in Stepper

Fixes: https://github.com/Automattic/wp-calypso/issues/97067

## Proposed Changes

* This fixes the navigation buttons and the formatted header in Stepper and makes them consistent with /start.
* It cleans up the never used `shouldStickyNavButtons` property and the CSS that comes with it.

## Why are these changes being made?

To bring consistency.

## Testing Instructions

1. Using incognito, open two tabs one to /setup/onboarding and one to /start.
2. The user step should look 99% identical between the two. **On desktop and on mobile.**
3. Feel free to leave incognito mode. 
5. The domain step should look 99% identical between the two. **On desktop and on mobile.**
6. The plans step should look 99% identical between the two. **On desktop and on mobile.**
